service/lib/config/config.auth.js

```diff
@@ -25,50 +15,50 @@ const resourcesConfig = new Reconfig({
 
 const Actions = {
   // organization
-  CreateOrganization: generateAction('organization', 'create'),
-  UpdateOrganization: generateAction('organization', 'update'),
-  ReadOrganization: generateAction('organization', 'read'),
-  DeleteOrganization: generateAction('organization', 'delete'),
-  ListOrganizations: generateAction('organization', 'list'),
-  AllOrganization: generateAction('organization'),
+  CreateOrganization: 'authorization:organizations:create',
```

Comment:
I did not want to change all the endpoints. I think that we can leave it like for now